### PR TITLE
pdksync - (MAINT) - Allow Stdlib 9.x

### PR DESCRIPTION
**Pull request by LukasAud**
(MAINT) - Update Stdlib dependency
pdk version: `2.5.0` 

Fixes https://github.com/puppetlabs/puppetlabs-lvm/issues/285

### CODE REVIEW

**Comment by jordanbreen28** [comment]
This may have to wait until CI fixes in https://github.com/puppetlabs/puppetlabs-lvm/pull/283 are merged.

**Review by bastelfreak — APPROVED** [review]

**Review by jordanbreen28 — APPROVED** [review]
LGTM pending ci

**Pull request merged**
